File: CS/DXTabControlExample/ViewModel/ContactViewModel.cs
using DevExpress.Mvvm.DataAnnotations;
using DXTabControlExample.Common;

namespace DXTabControlExample.ViewModel {
    [POCOViewModel]
    public class ContactViewModel {
        public virtual int Id { get; set; }
        public virtual Gender Gender { get; set; }
        public virtual string FirstName { get; set; }
        public virtual string LastName { get; set; }
        public virtual string CreditCardNumber { get; set; }
        public virtual string Email { get; set; }
        public virtual string Phone { get; set; }
        public virtual string Address { get; set; }
        public virtual string City { get; set; }
        public virtual string State { get; set; }
        public virtual string Zip { get; set; }
        public virtual byte[] Photo { get; set; }

        public ContactViewModel() { }
    }
}

File: VB/DXTabControlExample/ViewModel/ContactViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations
Imports DXTabControlExample.Common

Namespace DXTabControlExample.ViewModel
	<POCOViewModel>
	Public Class ContactViewModel
		Public Overridable Property Id() As Integer
		Public Overridable Property Gender() As Gender
		Public Overridable Property FirstName() As String
		Public Overridable Property LastName() As String
		Public Overridable Property CreditCardNumber() As String
		Public Overridable Property Email() As String
		Public Overridable Property Phone() As String
		Public Overridable Property Address() As String
		Public Overridable Property City() As String
		Public Overridable Property State() As String
		Public Overridable Property Zip() As String
		Public Overridable Property Photo() As Byte()

		Public Sub New()
		End Sub
	End Class
End Namespace
